frontend2/src/utils/api/team.ts

```diff
@@ -63,6 +64,22 @@ export const retrieveTeam = async (episodeId: string): Promise<TeamPrivate> => {
   return await API.teamTMeRetrieve({ episodeId });
 };
 
+/**
+ * Push a partial update to the current user's team for a specific episode.
+ * @param episodeId The episode of the team
+ * @param teamProfilePrivateRequest The partial team update
+ * @returns The updated TeamPrivate object
+ */
+export const updateTeamPartial = async (
+  episodeId: string,
+  teamProfilePrivateRequest: TeamProfilePrivateRequest,
```

Comment:
What is our standard practice for exposed API functions? I had thought we were listing parameters, i.e. `episodeId, teamId, userId, ...` instead of the whole `patchedTeamPrivateRequest`?

Reply:
I think if there are a small number of parameters, we can list them. `TeamProfilePrivateRequest` is an object with a lot of fields though, so its easier to take in a `TeamProfilePrivateRequest` than all the fields